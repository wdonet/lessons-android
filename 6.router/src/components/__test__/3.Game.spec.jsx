import React from 'react';
import renderer from 'react-test-renderer';
import Game from '../Game';

jest.mock('../Board.jsx', () => 'mock-board-tag')
jest.mock('../Winner.jsx', () => 'mock-winner-tag')

describe('<Game>', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<Game/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
