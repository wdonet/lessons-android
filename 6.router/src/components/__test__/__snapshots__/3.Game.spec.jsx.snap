// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Game> should render correctly 1`] = `
<div
  className="game"
>
  <div
    className="game-board"
  >
    <mock-board-tag
      onClick={[Function]}
      squares={
        Array [
          null,
          null,
          null,
          null,
          null,
          null,
          null,
          null,
          null,
        ]
      }
    />
  </div>
  <div
    className="game-info"
  >
    <div>
      Next player: X
    </div>
    <mock-winner-tag
      onReset={[Function]}
      winner={null}
    />
    <ol />
  </div>
</div>
`;
